config/crd/bases/spanner.mercari.com_spannerautoscalers.yaml

```diff
@@ -285,36 +285,40 @@ spec:
                     - targetServiceAccount
                     type: object
                   type:
-                    description: 'TODO: Add comments for describing each of the structs'
+                    enum:
+                    - gcp-sa-key
+                    - impersonation
+                    - adc
                     type: string
-                required:
-                - type
                 type: object
               scaleConfig:
                 properties:
                   computeType:
+                    enum:
+                    - nodes
+                    - processing-units
                     type: string
                   nodes:
                     properties:
                       max:
                         type: integer
                       min:
                         type: integer
-                    required:
-                    - max
-                    - min
                     type: object
                   processingUnits:
                     properties:
                       max:
+                        multipleOf: 100
                         type: integer
                       min:
+                        multipleOf: 100
                         type: integer
                     required:
                     - max
                     - min
                     type: object
                   scaledownStepSize:
+                    default: 2
```

Comment:
https://github.com/mercari/spanner-autoscaler/pull/65/files#diff-81d59bd0d0ec7fde640490ad3785a60212a04c950c8b1afec25ecfae1608666fR94-R96

2000?

Reply:
Sorry, this has been fixed in #66, along with all other units also converted to PU. Let me know if that is acceptable - https://github.com/mercari/spanner-autoscaler/pull/66/files#diff-62de20254f730d620b7d82abbdae5ac4dd77a6e7bcd873365265b2ee909ba798R345

Reply:
I see, thanks!